index.js

```diff
@@ -78,6 +105,29 @@ client.on(Events.InteractionCreate, interaction => {
 
         interaction.reply(invertedMessage);
     }
+
+    // Task creation slash command
+    if (interaction.commandName === 'addtask') {
+
+        const task = interaction.options.get('task').value;
+        const mirror = interaction.options.get('mirror').value; 
+        const user = mirror ? MIRROR_USER : usersMap.get(interaction.user.id); 
+
+        // Create task
+        const mailOptions = createTask(user, task);
```

Comment:
Tiny naming suggestion: what do you think about `getMailOptions()`? Since the actual creation happens later, that might make it a bit clearer at a glance for anyone reading through the code!